### PR TITLE
Comment-SBB20

### PR DESCRIPTION
**Pull request by Aurige**
Extension of ConnectionEnd (submode and transportOrganisation)

### CODE REVIEW

File: xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd [hunk 1]
```diff
@@ -561,14 +564,24 @@ Rail transport, Roads and Road transport
 		<xsd:sequence>
 			<xsd:element name="TransportMode" type="AllVehicleModesOfTransportEnumeration" default="all" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>MODE of end Point of TRANSFER . Default is all modes. of SCHEDULED STOP POINT Can be derived.</xsd:documentation>
+					<xsd:documentation>MODE of end Point of CONNECTION. Default is all modes,  MODE of SCHEDULED STOP POINT Can be derived.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd]: Here you change Transfer to Connection, but at line 572 it is again a Transfer.

Comment (Aurige) [reply xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd]: oups... corrected

**Comment by skinkie** [comment]
This change request does not have documentation.
image

**Comment by Aurige** [comment]
Ok, let's keep it with this "needs documentation update" and then we will manage all the ones with this label at once.

**Comment by skinkie** [comment]
image
image

@nick-knowles this should then also be done on new modes.

**Review by ue71603 — APPROVED** [review]

File: xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd [hunk 2]
```diff
@@ -561,12 +564,22 @@ Rail transport, Roads and Road transport
...
```
Comment (skinkie) [inline xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd:567]:
```suggestion
					<xsd:documentation>MODE of end Point of CONNECTION. Default is all modes, MODE of SCHEDULED STOP POINT can be derived.</xsd:documentation>
```

File: xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd [hunk 3]
```diff
@@ -561,12 +564,22 @@ Rail transport, Roads and Road transport
...
 			</xsd:element>
 			<xsd:element name="ScheduledStopPointRef" type="ScheduledStopPointRefStructure" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Identifier of end SCHEDULED STOP POINT of TRANSFER.</xsd:documentation>
+					<xsd:documentation>Identifier of end SCHEDULED STOP POINT of CONNECTION</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element ref="Submode" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>SUBMODE of end Point of CONNECTION. SUBMODE of SCHEDULED STOP POINT Can be derived. </xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd:577]:
```suggestion
					<xsd:documentation>SUBMODE of end Point of CONNECTION. SUBMODE of SCHEDULED STOP POINT can be derived.</xsd:documentation>
```

File: xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd [hunk 4]
```diff
@@ -561,12 +564,22 @@ Rail transport, Roads and Road transport
...
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element ref="TransportOrganisationRef" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>TransportOrganisation to which the end of CONNECTION applies. </xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_tacticalPlanning/netex_servicePattern_version.xsd:582]:
```suggestion
					<xsd:documentation>TransportOrganisation to which the end of CONNECTION applies.</xsd:documentation>
```

**Review by skinkie — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Pull request merged**